pilot/pkg/proxy/envoy/v2/ads.go

```diff
@@ -390,7 +390,7 @@ func (s *DiscoveryServer) StreamAggregatedResources(stream ads.AggregatedDiscove
 		adsLog.Warnf("Error reading config %v", err)
 		return err
 	}
-	if len(s.globalPushContext().Services(nil)) == 0 {
+	if s.globalPushContext().Services(nil) == nil {
```

Comment:
I dig into the code, we can never match this condition once we InitContext https://github.com/istio/istio/pull/10170/commits/3d3ef2f0b1338ecdf7427d6c945d11d24eaac4af#diff-14f4a8a239bf5ba82053cf6606daace5R386.